appveyor.yml

```diff
@@ -1,21 +1,24 @@
 cache:
   - node_modules
+  - "%LOCALAPPDATA%\\Yarn"
+
+init:
+  - git config --global core.autocrlf true
```

Comment:
@stof Could you explain what you mean?
I got this from the yarn website: https://yarnpkg.com/lang/en/docs/install-ci/#appveyor-tab
Looking at other projects they use the same line